### ISSUE TITLE
[Android] Could not find com.facebook.react:react:0.25.1

### ISSUE THREAD

**Issue by MayeuldP** [lelandrichardson/react-native-maps#263]
Hey guy's ! 

I get this error after doing installation setup of react-native-maps and with react-native run-android. 

I get : `Build failed with exception`
`Could not find com.facebook.react:react:0.25.1` required by `com.airbnb.android:react-native-maps:0.5.0`.

Any ideas or resolve links ? I don't know why facebook is involved here :/

**Comment by allthetime** [comment]
If you look at the native modules that react-native-maps calls (inside the android folder of this repo) they all require com.facebook.react, which was created by facebook. 

Anyway, do you have this line inside `dependencies` in YOUR_APP/android/app/build.gradle:

`compile "com.facebook.react:react-native:+"`

I just took it out in mine to check and it gives the same error you got. 

It should look kind of like this:

```
dependencies {

  //... (other stuff)

    compile "com.facebook.react:react-native:+" 
    compile 'com.airbnb.android:react-native-maps:0.5.0'
}
```

**Comment by MayeuldP** [comment]
Yeah I'm stupid. And i tried your code. It doesn't work, I still have the same error.

I added the dependencies you said, and there wasn't `    compile "com.facebook.react:react-native:+" 
` ... Is there anything that I could forget ? :/

**Comment by stefan-digital-chefs** [comment]
check this:
https://github.com/lelandrichardson/react-native-maps/pull/241

**Comment by devux** [comment]
@MayeuldP update your react-native version to 0.25.1 it will get solve

**Comment by MayeuldP** [comment]
Hi @devux , yes it's solved it, but I get this error : `Warning ReactNative.Component is deprecated. Use React.Component from the "react" package instead` . <br>I followed this topic http://stackoverflow.com/questions/37052798/reactnative-createclass-is-deprecated-use-react-createclass-from-react-package, but I still have the warning, I try `cd android/ && ./gradlew clean` and re-build my project, ...doesn't work. Any idea why ? And anyway <br> 
```
import MapView from 'react-native-maps';

```



  ```
renderScene() {
    var _this = this;
    if (!_this.user)
      return (
        <View style={styles.container}>
            <MapView
              initialRegion={{
                latitude: 37.78825,
                longitude: -122.4324,
                latitudeDelta: 0.0922,
                longitudeDelta: 0.0421,
              }}
            />
        </View>
      );} 
  };
```

display nothing on the screen (blank...). :/

**Comment by Leekao** [comment]
I'm using 0.26.3 and I get the same error:

Error:A problem occurred configuring project ':app'.
> Could not resolve all dependencies for configuration ':app:_debugCompile'.
   > Could not find com.facebook.react:react-native:0.25.1.
     Searched in the following locations:
         file:/C:/Program Files/Android/Android Studio2/gradle/m2repository/com/facebook/react/react-native/0.25.1/react-native-0.25.1.pom
         file:/C:/Program Files/Android/Android Studio2/gradle/m2repository/com/facebook/react/react-native/0.25.1/react-native-0.25.1.jar
         https://jitpack.io/com/facebook/react/react-native/0.25.1/react-native-0.25.1.pom
         https://jitpack.io/com/facebook/react/react-native/0.25.1/react-native-0.25.1.jar
         file:/C:/Users/Home/.m2/repository/com/facebook/react/react-native/0.25.1/react-native-0.25.1.pom
         file:/C:/Users/Home/.m2/repository/com/facebook/react/react-native/0.25.1/react-native-0.25.1.jar
         https://jcenter.bintray.com/com/facebook/react/react-native/0.25.1/react-native-0.25.1.pom
         https://jcenter.bintray.com/com/facebook/react/react-native/0.25.1/react-native-0.25.1.jar
         file:/D:/projects/ListenInApp/node_modules/react-native/android/com/facebook/react/react-native/0.25.1/react-native-0.25.1.pom
         file:/D:/projects/ListenInApp/node_modules/react-native/android/com/facebook/react/react-native/0.25.1/react-native-0.25.1.jar
         https://maven.fabric.io/public/com/facebook/react/react-native/0.25.1/react-native-0.25.1.pom
         https://maven.fabric.io/public/com/facebook/react/react-native/0.25.1/react-native-0.25.1.jar
         file:/C:/Users/Home/AppData/Local/Android/sdk/extras/android/m2repository/com/facebook/react/react-native/0.25.1/react-native-0.25.1.pom
         file:/C:/Users/Home/AppData/Local/Android/sdk/extras/android/m2repository/com/facebook/react/react-native/0.25.1/react-native-0.25.1.jar
         file:/C:/Users/Home/AppData/Local/Android/sdk/extras/google/m2repository/com/facebook/react/react-native/0.25.1/react-native-0.25.1.pom
         file:/C:/Users/Home/AppData/Local/Android/sdk/extras/google/m2repository/com/facebook/react/react-native/0.25.1/react-native-0.25.1.jar
     Required by:
         ListenIn:app:unspecified > com.airbnb.android:react-native-maps:0.5.0

**Comment by MayeuldP** [comment]
> update your react-native version to 0.25.1 it will get solve

Try to go back on this version, and it will solve it normally.... (maybe this package is doesn't support on new version > 0.25 ....)

**Comment by Hashan1100** [comment]
add repository

maven {
        url 'http://repo1.maven.org/maven2'
    }

and use 'com.facebook.react:react-native:0.20.1'
